Added packages python-collada and urdf2webots-pip to rosdep/python.yaml

DavidMansolino opened the pull request:
I need those two dependencies in my ros package `webots_ros2` (see PR cyberbotics/webots_ros2#28 that is waiting).

The goal is to be able to provide some utilities for ROS2 to be able to convert files between URDF (and XACRO in the furture) and PROTO, to ease the transition from Gazebo to Webots and the other way around so each one can use the simulator he prefers with ROS2 really easily.

Links:
  - **python-collada**
     - **debian**: https://packages.debian.org/search?keywords=python-collada
    - **ubuntu**: https://packages.ubuntu.com/search?keywords=python-collada&searchon=names
  - **python3-collada-pip**
     - **all**: https://pypi.org/project/pycollada/
  - **urdf2webots**
     - **all**: https://pypi.org/project/urdf2webots/

DavidMansolino commented:
@tfoote thank you for the constructive comment and pointers.

I think I have adressed all of them (description of the PR updated).

cottsay commented:
> ...please provide links to the research you've done to identify the packages...

We'd still like to see links to show that the package names are correct (again, see [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions)). Thanks!

DavidMansolino commented:
Sorry for the missing links.

- **python-collada**
   - **debian**: https://packages.debian.org/search?keywords=python-collada
   - **ubuntu**: https://packages.ubuntu.com/search?keywords=python-collada&searchon=names
   - **fedora**: https://apps.fedoraproject.org/packages/python-collada
- **urdf2webots**
   - **all**: https://pypi.org/project/urdf2webots/

cottsay commented:
Thanks, @DavidMansolino.

It looks like the Debian and Ubuntu packages are Python 2, and the Fedora package is Python 3. Which Python version do you want for `collada`?

Either way, they should be consistent. If you want python 2, you'll have to drop the Fedora rule, since Fedora is no longer packaging Python 2 packages.

DavidMansolino commented:
Actually, I would need both (as I will use them for ROS melodic and ROS Dashing). But that's fine if I only have Python2  for Fedora.
But about Python3 for Ubuntu and Debian, should I fallback to pip?

cottsay commented:
> Actually, I would need both...

Since you still want a `python-collada` that targets Python 2, you should just drop the Fedora rule from that key.

> ...about Python3 for Ubuntu and Debian, should I fallback to pip?

Since Ubuntu and Debian may someday get an official Python 3 package, I think it would be best to introduce a separate `python3-collada-pip` key (covering Debian, Fedora and Ubuntu).

DavidMansolino commented:
@cottsay thank you for your suggestions that perfectly make sense. I have modified the packages as you suggested and updated the links in the description of the PR.
